### DISCUSSION TITLE
I keep getting connection errors with the new server I installed today

### DISCUSSION THREAD

**Discussion by nanedi** [libre-tube/LibreTube#5758] [General]
### Steps to reproduce

1. Follow https://docs.piped.video/docs/self-hosting/
2. Installed a server with caddy
3. Add to 0.21.1
4. Loading the Feed
5. Loaded successfully
6. Get error notification

![Screenshot_20240317_145800_LibreTube](https://github.com/libre-tube/LibreTube/assets/3816371/1495991b-1371-46a2-91fe-0d1aa3bcf65c)

### Expected behavior

No error notification / alert.

### Actual behavior

Get error notification

### LibreTube version

0.21.1

### Android version

14

### Other details

New server has IPv4 and IPv6 at same time.

Frontend piped.xxxx.com
Backend pipedapi.xxxx.com
Proxy pipedproxy.xxxx.com

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
piped.xxxx.com loads fine and plays videos?

**Reply by nanedi** [reply]
Yes. Web also fine .

**Reply by nanedi** [reply]
I found the reason and it seems to be that the subscription list is not loading.

I have installed several other servers and they all have the same situation.

My 2 old servers are fine.

Web is fine. 

Plz check it, I think this is a bug.

**Comment by nanedi** [comment]
Opened an issue.

https://github.com/libre-tube/LibreTube/issues/5760

**Comment by nanedi** [comment]
I found the loading failed because some channels were updating their links or something. After manually re-subscribing to all the channels, everything works fine.
